test/cli/dbt-generate-profile/exists-invalid/expected-stderr

```diff
@@ -1,2 +1,3 @@
-Error: dbt profiles file "profiles.yml" is invalid:
+Error:
+- dbt profiles file "profiles.yml" is invalid:
```

Comment:
Povodne sa kratky prefix nezalamoval, no bolo to zbytocne komplikovane co sa tyka implementacia.